jwst/refpix/tests/test_refpix.py

```diff
@@ -0,0 +1,675 @@
+import numpy as np
+import pytest
+
+from jwst.datamodels import MIRIRampModel, dqflags
+from jwst.datamodels import RampModel, dqflags
+from jwst.refpix import RefPixStep
+from jwst.refpix.reference_pixels import Dataset, NIRDataset, correct_model, create_dataset
+
+
+def test_refpix_subarray():
+    '''Check that the correction is skipped for MIR subarray data '''
+
+    # For MIRI, no reference pixel correction is performed on subarray data
+    # No changes should be seen in the data arrays before and after correction
+
+    # create input data
+    # create model of data with 0 value array
+    ngroups = 50
```

Comment:
Probably not necessary to use that many groups just to test that no changes are made. Can keep data volume and test execution down by reducing this to something like only 5-10.

Reply:
True enough. That probably carried over from another test. 5 frames would be fine.

Reply:
Okay, I will go ahead and make the change.